sympy/solvers/pde.py

```diff
@@ -567,7 +572,17 @@ def pde_1st_linear_constant_coeff(eq, func, order, match, solvefun):
     where `a`, `b` and `c` are constants and `G(x, y)` can be an arbitrary
     function in `x` and `y`.
 
-    The general solution of the PDE is::
+    The general solution of the PDE is:
+
+    .. math::
+        f(x, y) = \left. \left[F(\eta) + \frac{1}{b^2 + c^2}
+        \int^{b x + c y} G\left (\frac{b \xi + c \eta}{b^2 + c^2},
+        \frac{- b \eta + c \xi}{b^2 + c^2} \right )
+        e^{\frac{a \xi}{b^2 + c^2}}\, d\xi\right)
+        e^{- \frac{a \xi}{b^2 + c^2}}
+        \right|_{\substack{ \eta=- b y + c x\\ \xi=b x + c y }}
```

Comment:
I think your formula is not aligned with the equation

<table><tr><td><img src="https://user-images.githubusercontent.com/34944973/53186658-c1496700-3644-11e9-9b4b-7cd9200f99de.png" width="512px"></td></tr></table>


You would have to change `b, c` to `a, b` like below

<table><tr><td><img src="https://user-images.githubusercontent.com/34944973/53186848-3026c000-3645-11e9-8d16-4e10db721fa7.png" width="512px"></td></tr></table>

I also find that the example section had not aligned coefficients with the 
Viktor Grigoryan, “Partial Differential Equations” Math 124A - Fall 2010, pp.7.
Can you also edit the coefficients in example?

Reply:
Done.

Reply:
<img src="https://user-images.githubusercontent.com/34944973/53244661-e4c8ec00-36ee-11e9-9351-6f1d3307444f.png" width="256px">

`e^{\frac{a \xi}{a^2 + b^2}}`  ->  `e^{\frac{c \xi}{a^2 + b^2}}`

Reply:
Done.